lms/static/certificates/sass/views/_certificate.scss

```diff
@@ -0,0 +1,250 @@
+      @extend %link-copy;
+    }
+  }
+
+  // status
+  .status {
+    @include row();
+    padding: $baseline-v ($baseline-h*2);
+    border-top-radius: ($baseline-v/10);
+    box-shadow: inset 0 -2px 2px 0 $shadow-l2;
+    background: $gray-l4;
+    color: $white-t;
+    text-transform: uppercase;
+    letter-spacing: 0.1rem;
+
+    @include media($bp-ds) {
+      padding: ($baseline-v) ($baseline-h);
+    }
+
+    @include media($bp-m) {
+      padding: ($baseline-v) ($baseline-h);
+    }
+
+    .title {
+      @extend %t-title-6;
+
+      @include media($bp-dl) {
+        @include span-columns(9 of 12);
+      }
+
+      @include media($bp-dm) {
+        @include span-columns(9 of 12);
+      }
+
+      @include media($bp-ds) {
+        @include font-size(14); // can't use %t-title-8 for some reason
+        @include lh(14);
+      }
+
+      @include media($bp-m) {
+        @include font-size(14); // can't use %t-title-8 for some reason
+        @include lh(14);
+      }
+    }
+  }
+
+  // supplemental content
+  .content-supplemental {
+
+    @include media($bp-dm) {
+    }
+
+    @include media($bp-ds) {
+    }
+
+    @include media($bp-m) {
+    }
+
+    .title {
+      @extend %t-title-7;
+      @extend %t-weight3;
+      margin-bottom: ($baseline-v/2);
+    }
+
+    .copy {
+      @extend %t-copy-sub1;
+      color: $gray-l2;
+    }
+
+    .list-actions {
+      margin-top: ($baseline-v);
+
+      .action {
+        @extend %t-copy-sub1;
+        @extend %btn-primary;
+
+        @include media($bp-dm) {
+
+        }
+
+        @include media($bp-ds) {
+          display: block;
+          text-align: center;
+        }
+
+        @include media($bp-m) {
+          display: block;
+          text-align: center;
+        }
+      }
+    }
+  }
+
+  .content-supplemental-how {
+    @extend %trans-size;
+    @include span-columns(3 of 12);
+
+    @include media($bp-dm) {
+      @include span-columns(6 of 12);
+    }
+
+    @include media($bp-ds) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+
+    @include media($bp-m) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+  }
+
+  .content-supplemental-certificates {
+    @extend %trans-size;
+    @include span-columns(3 of 12);
+    @include omega();
+
+    @include media($bp-dm) {
+      @include span-columns(6 of 12);
+    }
+
+    @include media($bp-ds) {
+      @include fill-parent;
+    }
+
+    @include media($bp-m) {
+      @include fill-parent;
+    }
+  }
+
+  .content-supplemental-about {
+    @extend %trans-size;
+    @include span-columns(6 of 12);
+
+    @include media($bp-dm) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+
+    @include media($bp-ds) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+
+    @include media($bp-m) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+  }
+
+  // TODO - Determine what basic types are supported
```

Comment:
A follow up story in the backlog currently exists to define templates for openedX equivalents to our honor (no signature or watermark) and verified certificates (watermark and signatures), and to also have views for all 4 of our edX specific certificates. In this work I think it makes sense to think through abstraction and the theming considerations you outlined below.  

I think this is ok to keep in for now.